core/tests/protractor_desktop/explorationTranslationTab.js

```diff
@@ -21,41 +21,33 @@ var forms = require('../protractor_utils/forms.js');
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
+var CreatorDashboardPage = require(
+  '../protractor_utils/CreatorDashboardPage.js');
 
 
 var ExplorationEditorPage =
   require('../protractor_utils/ExplorationEditorPage.js');
 
-describe('Exploration translation', function() {
+describe('Exploration translation and voiceover tab', function() {
+  var creatorDashboardPage = null;
   var explorationEditorMainTab = null;
   var explorationEditorPage = null;
   var explorationEditorSettingsTab = null;
   var explorationEditorTranslationTab = null;
+  var YELLOW_STATE_PROGRESS_COLOR = 'rgb(233, 179, 48)';
+  var GREEN_STATE_PROGRESS_COLOR = 'rgb(209, 72, 54)';
+  var RED_STATE_PROGRESS_COLOR = 'rgb(22, 167, 101)';
 
-  beforeEach(function() {
+  beforeAll(function() {
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
```

Comment:
Can you explain how this is supposed to work here?

Reply:
I'm creating an instance of CreatorDashboardPage, which I'll use later for opening exploration editor page!